### PR TITLE
More fixes for infinite scroll

### PR DESCRIPTION
**Pull request by hamstu**
### Purpose

Ignore whitespace in the PR as there is some formatting in there

#### Staging Deployment
To visit the URL of the staging deployment, please click "Show all checks" at the bottom of this PR and click "details" next to `bufferbotbrains/cicd-buffer-publish-legacy`

### CODE REVIEW

File: packages/profile-page/index.js [hunk 1]
```diff
@@ -6,62 +6,71 @@ import { actions as dataFetchActions } from '@bufferapp/async-data-fetch';
...
-        default:
-          requestName = 'queued';
+export default hot(
+  connect(
+    (state, ownProps) => {
+      const { tabId, profileId, childTabId } =
+        getProfilePageParams({ path: ownProps.history.location.pathname }) ||
+        {};
+      // With analytics, the reducer state name doesnt match the tabId
+      const reducerName =
```
Comment (albennett) [inline packages/profile-page/index.js]: 💅

Comment (hamstu) [reply packages/profile-page/index.js]: Fun fact, I just used the 'Prettier' extension in VS Code to format this file! Just something I've been experimenting with. It's not set to auto-format on save or anything, but it's nice in a pinch when the file is getting complex. :)

File: packages/queue/middleware.js [hunk 2]
```diff
@@ -91,19 +91,6 @@ export default ({ dispatch, getState }) => next => (action) => {
         notificationType: 'success',
         message: 'Okay, we\'ve deleted that post!',
       }));
-      /**
```
Comment (albennett) [inline packages/queue/middleware.js]: Do we no longer need to re-fetch because of the react hooks?

Comment (hamstu) [reply packages/queue/middleware.js]: We **do** need to re-fetch, but I moved the re-fetching further down in the middleware in a previous PR. Removing it here was part of resolving a merge conflict where it came back, I think.

Let me know if that makes sense! :)

**Pull request merged**